src/net_processing.cpp

```diff
@@ -3525,15 +3525,18 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         const ReconciliationRegisterResult result = m_txreconciliation->RegisterPeer(pfrom.GetId(), pfrom.IsInboundConn(),
                                                                                 peer_txreconcl_version, remote_salt);
         switch (result) {
-            case ReconciliationRegisterResult::NOT_FOUND:
+            case ReconciliationRegisterResult::NOT_FOUND: {
+                LogPrintLevel(BCLog::NET, BCLog::Level::Debug, "Ignore unexpected txreconciliation signal from peer=%d\n", pfrom.GetId());
+                break;
+            }
             case ReconciliationRegisterResult::SUCCESS:
                 break;
             case ReconciliationRegisterResult::ALREADY_REGISTERED:
                 LogPrintLevel(BCLog::NET, BCLog::Level::Debug, "txreconciliation protocol violation from peer=%d (sendtxrcncl received from already registered peer); disconnecting\n", pfrom.GetId());
                 pfrom.fDisconnect = true;
                 return;
             case ReconciliationRegisterResult::PROTOCOL_VIOLATION:
-                LogPrintLevel(BCLog::NET, BCLog::Level::Debug, "txreconciliation protocol violation from peer=%d; disconnecting\n", pfrom.GetId());
+            LogPrintLevel(BCLog::NET, BCLog::Level::Debug, "txreconciliation protocol violation from peer=%d; disconnecting\n", pfrom.GetId());
```

Comment:
nit, if you touch again: indentation changes for the worse here.  Also, usually while case block is indented, the `case` lines aren't indented wrt the `switch` line, see other examples or apply clang-format script.